src/main/webapp/downloads/beta.jsp

```diff
@@ -12,7 +12,7 @@
 
     <div id="ignite_subnav">
         <ul>
-            <li id="subnav01"><a href="index.jsp">Releases</a></li>
+            <li id="subnav01"><a href="./">Releases</a></li>
```

Comment:
Usage of `./` makes me wonder give this [stackoverflow answer](https://stackoverflow.com/questions/296873).  I am ignorant on the distinction though.  Otherwise, this PR looks great, thanks!